Formula/libcdio-paranoia.rb

```diff
@@ -0,0 +1,22 @@
+class LibcdioParanoia < Formula
+  desc "Audio CD ripper based on libcdio"
+  homepage "https://www.gnu.org/software/libcdio/"
+  url "https://ftp.gnu.org/gnu/libcdio/libcdio-paranoia-10.2+0.94+2.tar.gz"
```

Comment:
What is the version number here? How does it work?

Reply:
I am not the author of the program. I am only providing a formula for the package to build under home brew. I am not sure how their versioning work.

Reply:
I did some sleuthing, and it looks like 10.2 refers to the CDDA Paranoia version, 0.94 refers to the `libcdio` version, and `+2` is some kind of revision number.